openslides_backend/http/http_exceptions.py

```diff
@@ -1,5 +1,5 @@
 import json
-from typing import Any, Dict, List, Optional, Tuple
+from typing import Any, Dict, List, Optional, Text, Tuple
```

Comment:
looking at https://docs.python.org/3/library/typing.html#typing.Text, it seems that we should use `str` since we did not migrate from python2, or did werkzeug changed its type annotations?

Reply:
Yes, the latter. The correct type would be WSGIEnvironment, but since I do not seem to have access to that, I will stick to their analog typedef.